react-native-libraries.json

```diff
@@ -12008,5 +12008,17 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/dominicstop/react-native-ios-visual-effect-view",
+    "examples": [
+      "https://github.com/dominicstop/react-native-ios-visual-effect-view/tree/main/example/src/examples"
+    ],
+    "images": [
+      "https://github.com/dominicstop/react-native-ios-visual-effect-view/raw/main/assets/example-demo-01.gif"
+    ],
+    "ios": true,
+    "newArchitecture": true,
+    "newArchitectureNote": "Support for both the new architecture (fabric), and backward compat. to the old architecture (paper) is handled via react-native-ios-utilities"
```

Comment:
```suggestion
```

I think that noting backward compatibility with legacy code is not that important. The idea of that field was to describe the limitations, if New Architecture is not supported fully.

Reply:
okie